lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1111,7 +1112,50 @@ static bool ParseLangArgs(LangOptions &Opts, ArgList &Args,
   Opts.EnableIndexingSystemModuleRemarks = Args.hasArg(OPT_remark_indexing_system_module);
 
   Opts.EnableSkipExplicitInterfaceModuleBuildRemarks = Args.hasArg(OPT_remark_skip_explicit_interface_build);
-  
+
+  if (Args.hasArg(OPT_enable_library_evolution)) {
+    Opts.SkipNonExportableDecls |=
+    Args.hasArg(OPT_experimental_skip_non_exportable_decls);
+
+    Opts.SkipNonExportableDecls |=
+    Args.hasArg(OPT_experimental_skip_non_inlinable_function_bodies) &&
+    Args.hasArg(
+                OPT_experimental_skip_non_inlinable_function_bodies_is_lazy);
```

Comment:
Nitpick: it looks like the move lost the indention for these few lines.

Reply:
good catch